docs/tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver.rst

```diff
@@ -185,11 +194,11 @@ Argo workflow duration
 +------------+------------------+---------------+
 | data scale | without vineyard | with vineyard |
 +============+==================+===============+
-| 8500 Mi    | 186s             | 169s          |
+| 8500 Mi    | 189s             | 164s          |
 +------------+------------------+---------------+
-| 12000 Mi   | 250s             | 203s          |
+| 12000 Mi   | 234s             | 199s          |
 +------------+------------------+---------------+
-| 15000 Mi   | 332s             | 286s          |
+| 15000 Mi   | 252s             | 298s          |
 +------------+------------------+---------------+
```

Comment:
Why there are so huge improvements on "without vineyard"? and "with vineyard" becomes slower than "without vineyard"?

Reply:
Sorry, wrong writing here.